include/dali/core/tensor_view.h

```diff
@@ -481,6 +481,22 @@ struct TensorListView<Backend, DataType, DynamicDimensions>
   TensorListView &operator=(const TensorListView &) = default;
   TensorListView &operator=(TensorListView &&) = default;
 
+  /** @{ */
+  /** @brief Construction from contiguous memory */
+
+  TensorListView(std::nullptr_t, const std::vector<TensorShape<DynamicDimensions>> &shapes)
+      : Base(TensorListShape<DynamicDimensions>(shapes)) {}
+
+  template <int other_sample_ndim>
+  TensorListView(std::nullptr_t, const TensorListShape<other_sample_ndim> &shape)
+      : Base(shape) {}
+
+  template <int other_sample_ndim>
+  TensorListView(std::nullptr_t, TensorListShape<other_sample_ndim> &&shape)
+      : Base(std::move(shape)) {}
+
+  /** @} */
+
```

Comment:
These constructors already exist in `static_dim` flavour of `TensorListView`. Here I'm adding them to `DynamicDimensions` flavour as well.